config-templates/redhat/odf-local/4.8/custom-parameters.json

```diff
@@ -100,6 +119,10 @@
     "displayname": "Perform Cleanup",
     "name": "perform-cleanup",
     "default": "false",
+    "regex": "^(?:essentials|advanced)$",
+    "min-length": "8",
+    "max-length": "10",
+    "supported-values": ["essentials", "advanced"],
```

Comment:
same here

Reply:
Addressed in https://github.com/IBM/ibm-satellite-storage/pull/194/commits/bfad1599711afef7153182a023ded18ea5e91331